Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN \
     tr -d " \t\n\r" | sed -e 's/usr/|usr/g' -e 's/^.//') && \
   INSTALLED=$(apk info -q -L mariadb-common mariadb linux-pam | grep "\S") && \
   for path in $(echo "${INSTALLED}" | grep -v -E "${TO_KEEP}"); do \
+    echo "remove ${path}";\
```

Comment:
The changes to the `Dockerfile` looks unrelated to the compose introduction. Are they needed?

Reply:
I was failed to upgrade mariadb version. I think it's necessary to know which files was deleted during the Docker-image-building process.